### PR TITLE
clean up scripts

### PR DESCRIPTION
**Pull request by pweil-**
@jim-minter @kwoodson @kargakis 

Remove most of the scripts and vars in the template that they used.  Only thing in the bastion now is making sure the pub/private keys are available so you can ssh to the other nodes.

### CODE REVIEW

File: parts/openshift/3.7/bastion.json [hunk 1]
```diff
@@ -47,59 +38,14 @@
         "description" : "imageReference object that should be used to create the virtual machine"
       }
     },
-    "RHNUserName" : {
```
Comment (kargakis) [inline parts/openshift/3.7/bastion.json]: yay!

File: parts/openshift/vars.t [hunk 2]
```diff
@@ -1,5 +1,5 @@
-    "gituser" : "kargakis",
-    "branch" : "master",
+    "gituser" : "pweil-",
```
Comment (kargakis) [inline parts/openshift/vars.t]: gotcha

**Comment by kargakis** [comment]
fix the gituser x branch combo (again) and this lgtm

File: parts/openshift/vars.t (hunk 2 above)
Comment (pweil-) [reply parts/openshift/vars.t]: heh, just added that back in to test a new image with @kwoodson thought I could beat you to it :rofl:

**Comment by pweil-** [comment]
> fix the gituser x branch combo (again) and this lgtm

image test didn't work.  Removed the branch settings, ready for tagging

**Comment by pweil-** [comment]
/hold
hold this until we know where we're going with the work from @jim-minter.  He is using some pieces of the refarch config that is output.

**Comment by kargakis** [comment]
Cleaned up already 

/close

**Pull request closed without merging**